src/core/pointcloud/qgseptdecoder.cpp

```diff
@@ -466,4 +477,27 @@ QgsPointCloudBlock *QgsEptDecoder::decompressLaz( const QString &filename,
          );
 }
 
+#include <QDebug>
+
+QgsPointCloudBlock *QgsEptDecoder::decompressLaz( const QByteArray &byteArrayData,
+    const QgsPointCloudAttributeCollection &attributes,
+    const QgsPointCloudAttributeCollection &requestedAttributes )
+{
+  QString filename = "/tmp/node.txt";
```

Comment:
?

Reply:
I just changed this to use QDir::tempPath() + QDir::separator(). In this part I'm writing the downloaded data to a file and then that file content is decompressed. Any suggession how to make this better ?

Reply:
QDir::tempPath() is ok